### ISSUE TITLE
pre-built binaries does not show version

### ISSUE THREAD

**Issue by WindzCUHK** [enix/x509-certificate-exporter#33]
as title

setup
```bash
wget https://github.com/enix/x509-certificate-exporter/releases/download/v3.0.0/x509-certificate-exporter-linux-amd64
chmod +x ./x509-certificate-exporter-linux-amd64
```

current result
```console
$ ./x509-certificate-exporter-linux-amd64 --version
version 0.0.0
```

expected result
```console
$ ./x509-certificate-exporter-linux-amd64 --version
version 3.0.0
```

OS
```console
$ lsb_release -a
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	CentOS
Description:	CentOS Linux release 7.9.2009 (Core)
Release:	7.9.2009
Codename:	Core
```

**Comment by WindzCUHK** [comment]
## sidetrack
I think the module should be `v3` instead of `v2`

https://github.com/enix/x509-certificate-exporter/blob/0c6a3282b9f85953529384d8b6e740be0bbff73b/go.mod#L1

**Comment by arcln** [comment]
Hello, this is fixed in 3.0.1, thanks for your feedback.
